x-pack/plugins/security_solution/public/common/components/header_global/index.tsx

```diff
@@ -1,155 +0,0 @@
-import { useGetUrlSearch } from '../navigation/use_get_url_search';
-import { useGetUserCasesPermissions, useKibana } from '../../lib/kibana';
-import { APP_ID, ADD_DATA_PATH, APP_DETECTIONS_PATH } from '../../../../common/constants';
-import { useGlobalHeaderPortal } from '../../hooks/use_global_header_portal';
-import { LinkAnchor } from '../links';
-
-const Wrapper = styled.header<{ $isFixed: boolean }>`
-  ${({ theme, $isFixed }) => `
-    background: ${theme.eui.euiColorEmptyShade};
-    border-bottom: ${theme.eui.euiBorderThin};
-    width: 100%;
-    z-index: ${theme.eui.euiZNavigation};
-    position: ${$isFixed ? 'fixed' : 'relative'};
-  `}
-`;
-Wrapper.displayName = 'Wrapper';
-
-const WrapperContent = styled.div<{ $globalFullScreen: boolean }>`
-  display: ${({ $globalFullScreen }) => ($globalFullScreen ? 'none' : 'block')};
-  padding-top: ${({ $globalFullScreen, theme }) =>
-    $globalFullScreen ? theme.eui.paddingSizes.s : theme.eui.paddingSizes.m};
-`;
-
-WrapperContent.displayName = 'WrapperContent';
-
-const FlexItem = styled(EuiFlexItem)`
-  min-width: 0;
-`;
-FlexItem.displayName = 'FlexItem';
-
-const FlexGroup = styled(EuiFlexGroup)<{ $hasSibling: boolean }>`
-  ${({ $hasSibling, theme }) => `
-    border-bottom: ${theme.eui.euiBorderThin};
-    margin-bottom: 1px;
-    padding-bottom: 4px;
-    padding-left: ${theme.eui.paddingSizes.l};
-    padding-right: ${theme.eui.paddingSizes.l};
-    ${$hasSibling ? `border-bottom: ${theme.eui.euiBorderThin};` : 'border-bottom-width: 0px;'}
-  `}
-`;
-FlexGroup.displayName = 'FlexGroup';
-
-interface HeaderGlobalProps {
-  hideDetectionEngine?: boolean;
-  isFixed?: boolean;
-}
-
-export const HeaderGlobal = React.memo(
-  forwardRef<HTMLDivElement, HeaderGlobalProps>(
-    ({ hideDetectionEngine = false, isFixed = true }, ref) => {
-      const { globalHeaderPortalNode } = useGlobalHeaderPortal();
-      const { globalFullScreen } = useGlobalFullScreen();
-      const { timelineFullScreen } = useTimelineFullScreen();
-      const search = useGetUrlSearch(navTabs.overview);
-      const { application, http } = useKibana().services;
-      const { navigateToApp, getUrlForApp } = application;
-      const overviewPath = useMemo(
-        () => getUrlForApp(APP_ID, { path: SecurityPageName.overview }),
-        [getUrlForApp]
-      );
-      const overviewHref = useMemo(() => getAppOverviewUrl(overviewPath, search), [
-        overviewPath,
-        search,
-      ]);
-
-      const basePath = http.basePath.get();
-      const goToOverview = useCallback(
-        (ev) => {
-          ev.preventDefault();
-          navigateToApp(`${APP_ID}:${SecurityPageName.overview}`, { path: search });
-        },
-        [navigateToApp, search]
-      );
-
-      const hasCasesReadPermissions = useGetUserCasesPermissions()?.read;
-
-      // build a list of tabs to exclude
-      const tabsToExclude = new Set<string>([
-        ...(hideDetectionEngine ? [SecurityPageName.detections] : []),
```

Comment:
@XavierM did not see this used or called anywhere?

Reply:
good, it was an old Feature flag fro 7.5